src/runtime/SandBox.cpp

```diff
@@ -123,6 +123,9 @@ SandBox::SandBoxResult SandBox::run(const std::function<Value()>& scriptRunner)
 void SandBox::createStackTraceData(StackTraceDataVector& stackTraceData, ExecutionState& state)
 {
     ExecutionState* pstate = &state;
+#ifdef ESCARGOT_DEBUGGER
+    uint32_t i = 0;
+#endif /* ESCARGOT_DEBUGGER */
```

Comment:
I would add a newline here, and call `i` (usually represents index) as `executionStateDepth`.